app.js

```diff
@@ -0,0 +1,31 @@
+if (process.env.NODE_ENV !== 'production'){
```

Comment:
sebaiknya folder server dijadikan 1 folder server, terpisah dengan client

Reply:
ini juga biasanya kalo development yang menggunakan dotenv :D

Reply:
https://stackoverflow.com/questions/52546426/is-module-dotenv-for-development-only